lib/main.ts

```diff
@@ -275,7 +275,19 @@ export class Transpiler {
   private normalizeSlashes(path: string) { return path.replace(/\\/g, '/'); }
 
   private translateComment(comment: string): string {
-    return comment.replace(/\{@link ([^\}]+)\}/g, '[$1]');
+    comment = comment.replace(/\{@link ([^\}]+)\}/g, '[$1]');
+
+    // Remove the following tags and following comments till end of line.
+    comment = comment.replace(/@param.*$/gm, '');
```

Comment:
opt: what about just chaining the .replace calls instead of reassigning to the comment variable?

return comment.replace(/1/).replace(/2/).replace...
unless having separate statements is essential for stepping with a debugger?